lib/cylc/cfgvalidate.py

```diff
@@ -25,7 +25,7 @@
 from isodatetime.data import Calendar, Duration, TimePoint
 from isodatetime.dumpers import TimePointDumper
 from isodatetime.parsers import DurationParser, TimePointParser
-from parsec.validate import ParsecValidator, IllegalValueError
+from cylc.parsec.validate import ParsecValidator, IllegalValueError
```

Comment:
Can now combine this module with `cylc.parsec.validate`.

Reply:
Which one should stay, and which one should go?

Reply:
`cylc.parsec.validate` can probably absorb `cylc.cfgvalidate`.

Reply:
Done! Looks simpler now I think. Had to update some unit tests and a comment as well.